integrations/astrovault-adapter/src/dex.rs

```diff
@@ -0,0 +1,1113 @@
+use crate::ASTROVAULT;
+use crate::AVAILABLE_CHAINS;
+use abstract_dex_standard::Identify;
+use abstract_sdk::core::objects::PoolType;
+
+use abstract_sdk::feature_objects::AnsHost;
+
+#[derive(Default)]
+pub struct Astrovault {
+    pub pool_type: Option<PoolType>,
+}
+
+impl Identify for Astrovault {
```

Comment:
Can't we update the `Identity` trait to contain consts? Like this: https://doc.rust-lang.org/reference/items/associated-items.html#associated-constants

The function implementations can then be global.

Reply:
Yeah, we could do that! Created an issue on Linear